package.json

```diff
@@ -9,21 +9,18 @@
     "lint:scss": "postcss -c postcss_lint.json",
     "lint:js": "eslint .",
     "lint": "npm run lint:scss && npm run lint:js",
-
```

Comment:
Was this deliberate or just automatic editor behaviour? I’d prefer to keep this section chunked as it’s so long; if necessary add blank comments.